VIM-4712: Adding action sheet option to add to channel

ghking opened the pull request:
Companion PR to https://github.vimeows.com/MobileApps/Vimeo-iOS/pull/1324

nicolelehrer commented:
@ghking why are we removing `hasCopyrightMatch`? Could we do a quick PR description to account for these changes?

ghking commented:
@nicolelehrer Copyright match was an old feature we removed a while ago, so this function just must have not been removed. We weren't using it anywhere so I just removed it while I was in there.

cameo-github commented:
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><strong><a href="https://github.com/vimeo/VimeoNetworking/blob/81104af818e9558a3591ae902e87e2246b7f6e72/VimeoNetworking/Sources/VimeoClient.swift#L326">VimeoNetworking/Sources/VimeoClient.swift#L326</a></strong>: cast from ‘Response&lt;VIMNullResponse&gt;’ to unrelated type ‘Response&lt;ModelType&gt;’ always fails  <br /><code>
                let response = Response(model: nullResponseObject, json: [:]) as! Response&lt;ModelType&gt;
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 22 tests, with 0 failures (0 unexpected) in 0.361 (0.378) seconds</td>
    </tr>
  </tbody>
</table>

See build details on [CircleCI](https://circleci.com/gh/vimeo/VimeoNetworking/326)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>
